git_machete/cmd.py

```diff
@@ -1285,17 +1285,18 @@ def log(branch: str) -> None:
     run_git("log", "^" + fork_point(branch, use_overrides=True), f"refs/heads/{branch}")
 
 
-Commit = Tuple[str, str, str]
+Hash_ShortHash_Message = Tuple[str, str, str]
 
 
-def commits_between(earliest_exclusive: str, latest_inclusive: str) -> List[Commit]:
+def commits_between(earliest_exclusive: str, latest_inclusive: str) -> List[Hash_ShortHash_Message]:
     # Reverse the list, since `git log` by default returns the commits from the latest to earliest.
     return list(reversed(list(map(
         lambda x: x.split(":", 2),  # type: ignore
         non_empty_lines(popen_git("log", "--format=%H:%h:%s", f"^{earliest_exclusive}", latest_inclusive, "--"))
     ))))
 
 
+# TODO (#117): extract to namespace, use mypy
```

Comment:
namespace or enum? 🤔 or is "namespace" a word for enum?

Reply:
possibly my bad wording. I meant that after the changes it will be something like `SYNC_STATUS(what I meant to be the "namespace").UNTRACKED`